audb/core/load_to.py

```diff
@@ -464,13 +471,20 @@ def load_to(
 
     # save database and PKL tables
 
-    db.save(
-        db_root,
-        storage_format=audformat.define.TableStorageFormat.PICKLE,
-        update_other_formats=False,
-        num_workers=num_workers,
-        verbose=verbose,
-    )
+    if pickle_tables:
```

Comment:
This ìf...else` is the hardest bit to comprehend for me. 

I think this is the rationale:

- when pickling, you do not want to update the other formats but rather store every table as pickle.  As there are most often several tables, you want to forward the `num_workser`.
- when not pickling: you only want to manipulate the single header file as indicated hy `header_only`, and the number of workers can stay at the default of only one worker as multiprocessing is useless.

Should maybe a comment be added that motivates the  different choices of the parameters to the `db.save` call?

Reply:
I added the following comments:

```python
    if pickle_tables:
        # Store database header,
        # and add table as pickle files
        # (tables are already stored in their original format).
        # Uses `num_workers` to save tables in parallel.
        db.save(
            db_root,
            storage_format=audformat.define.TableStorageFormat.PICKLE,
            update_other_formats=False,
            num_workers=num_workers,
            verbose=verbose,
        )
    else:
        # Store database header
        # (tables are already stored)
        db.save(
            db_root,
            header_only=True,
            verbose=verbose,
        )
```